ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -6,7 +6,12 @@ set -o pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
-export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ ${ENABLE_SHARED_PHZ} == "yes" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred_shared_account"
```

Comment:
What are the contents of `.awscred_shared_account`? Is it the credentials for the other CI account; so that if you're using the `openshift-ci` account, `awscred_shared_account` contains the credentials for  `openshift-ci-2`?

Reply:
Right, Account A(VPC and PHZ owner)'s credential is saved in  `.awscred_shared_account`:
```
[default]
aws_access_key_id = ACCOUNT-A-KEY
aws_secret_access_key = ACCOUNT-A-SEC
aws_account_id = ACCOUNT-A-ID
```

Another way to handle this is to save all credentials in one `.awscred` file, like:
```
[default]
aws_access_key_id = ACCOUNT-B-KEY
aws_secret_access_key = ACCOUNT-B-SEC
aws_account_id = ACCOUNT-B-ID


[sharedaccount]
aws_access_key_id = ACCOUNT-A-KEY
aws_secret_access_key = ACCOUNT-A-SEC
aws_account_id = ACCOUNT-A-ID
```

Personally, I prefer to use different files is because the original `.awscred` file in the cluster profile won't be touched.